content/manuals/engine/network/packet-filtering-firewalls.md

```diff
@@ -119,6 +119,11 @@ the source and destination. For instance, if the Docker host has addresses
 `2001:db8:1111::2` and `2001:db8:2222::2`, you can make rules specific to
 `2001:db8:1111::2` and leave `2001:db8:2222::2` open.
 
+If your containers are also querying DNS, you should add this rule as well to allow them to work:
+```
+$ iptables -I DOCKER-USER -m state --state RELATED,ESTABLISHED -j ACCEPT
+```
```

Comment:
If you're updating, can you also;

- add a  newline before  the code-block
- add a `console` code-hint to  make  sure it's properly highlighted?


````suggestion
```console
$ iptables -I DOCKER-USER -m state --state RELATED,ESTABLISHED -j ACCEPT
```
````